test/cql-pytest/test_filtering.py

```diff
@@ -453,3 +453,11 @@ def test_multi_column_relation_wrong_number_of_items_in_tuple(cql, test_keyspace
             cql.execute(f'SELECT * FROM {table} WHERE a = 0 AND (b, c, d) IN ((1, 2), (2, 1, 4))')
         with pytest.raises(InvalidRequest, match='Invalid list literal for in'):
             cql.execute(f'SELECT * FROM {table} WHERE a = 0 AND (b, c, d) IN ((1, 2, 3, 4, 5), (2, 1, 4))')
+
+# Check that restricting regular column isn't ignored when selecting a static row.
+# Reproduces #20485
```

Comment:
I believe that issue 20485 is a duplicate of the pre-existing #10357, and this test is an almost identical duplicate of the existing test test/cql-pytest/test_static.py::test_filter_with_only_static.

Reply:
It is.